src/Cassandra/Policies/DCAwareRoundRobinPolicy.cs

```diff
@@ -23,19 +23,26 @@
 namespace Cassandra
 {
     /// <summary>
-    ///  A data-center aware Round-robin load balancing policy. <p> This policy
-    ///  provides round-robin queries over the node of the local datacenter. It also
-    ///  includes in the query plans returned a configurable number of hosts in the
-    ///  remote datacenters, but those are always tried after the local nodes. In
-    ///  other words, this policy guarantees that no host in a remote datacenter will
-    ///  be queried unless no host in the local datacenter can be reached. </p><p> If used
-    ///  with a single datacenter, this policy is equivalent to the
-    ///  <see cref="RoundRobinPolicy"/> policy, but its GetDatacenter awareness
-    ///  incurs a slight overhead so the <see cref="RoundRobinPolicy"/>
-    ///  policy could be preferred to this policy in that case.</p>
+    /// A data-center aware Round-robin load balancing policy.
+    /// <para>
+    /// This policy provides round-robin queries over the node of the local datacenter. Currently, it also includes in the query plans
+    /// returned a configurable number of hosts in the remote datacenters (which are always tried after the local nodes)
+    /// but this functionality will be removed in the next major version of the driver.
+    /// See the comments on <see cref="DCAwareRoundRobinPolicy(string, int)"/> for more information.
+    /// </para>
+    /// <para>
+    /// If used with a single datacenter, this policy is equivalent to the <see cref="RoundRobinPolicy"/> policy,
```

Comment:
I think its a good opportunity to remove this last paragraph, as it doesn't make much sense.

Reply:
👍 done